website/docs/d/dynamodb_table.html.markdown

```diff
@@ -0,0 +1,30 @@
+---
+layout: "aws"
+page_title: "AWS: aws_dynamodb_table"
+sidebar_current: "docs-aws-datasource-dynamodb-table"
+description: |-
+  Provides a DynamoDB table data source.
+---
+
+# aws_elb
```

Comment:
Nitpick: wrong data source name

Reply:
Attack of the copypasta! 😄 update incoming